scania-digital-design-system/sdds
# Feature - Refactor scss structure in theme package

**Describe pull-request**  
_Describe what the pull-request is about_
Quite big PR mostly related with cleaning up SCSS that aren't used anymore and moved it to an "old folder". Breaking changes in the core structure. 
- Move old SCSS to the old folder
- Cleaned up imports that aren't valid anymore
- Updated imports in current components
- Removed some IE-related solutions from gulp.

**Solving issue**  
_In case of GitHub issue add # plus the number of the issue (for example #123) OR if it is Azure then AB# and number of ticket_
Fixes: [AB#1144](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1144)

**How to test**  
_Add description of how to test if possible_
1. Setup a clean clone
2 npm install and build for all packages
3. Startup storybook (make sure it looks and works the same as master)
4. Try out the packages in different frameworks in the demo folder
5. Everything should look the same in all frameworks, Only old SCSS has been moved and should not affect the current styling and functionality 

Setting at draft for now
